AutolayoutPractice/RemindersList/Views/RemindersListViewController.swift

```diff
@@ -8,7 +8,13 @@
 import Foundation
 import UIKit
 
-class RemindersListViewController: UIViewController {
+class RemindersListViewController: UIViewController, AddReminderDelegate {
+    func addReminder(title: String, description: String, dueDate: Date) {
```

Comment:
i was initially going to make the viewmodel the delegate, but decided against in order to have more access/control over the tableView (reloadData), thoughts?

Reply:
Let's discuss today! I think if your view model is the data source it would make sense to have the view controller act as a delegate, but there are different approaches to consider as well